src/js/me-i18n-locale-en.js

```diff
@@ -33,7 +33,7 @@
             'mejs.fullscreen' : 'Fullscreen',
 
             // mep-feature-jumpforward
-            'mejs.time-jump-forward': 'Jump forward %1 seconds',
+            'mejs.time-jump-forward': 'Jump forward {0} {0|plural:second:seconds}',
```

Comment:
I changed this to `Jump forward 1 second;Jump forward %1 seconds`; that way we'll concatenate all the possible translations in the order dictated on http://localization-guide.readthedocs.io/en/latest/l10n/pluralforms.html